client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -321,6 +333,31 @@ class DeltaSharingRestClient(
     )
   }
 
+  private def getFilesByPage(
+      targetUrl: String,
+      request: QueryTableRequest): (Long, String, Seq[String]) = {
+    val allLines = ArrayBuffer[String]()
```

Comment:
consider adding a log message with a todo to remove it once the feature has been out for a bit.